libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -189,6 +224,10 @@ export class RushXCommandLine {
           quiet = true;
         } else if (argValue === '-h' || argValue === '--help') {
           help = true;
+        } else if (argValue === '-d' || argValue === '--debug') {
+          isDebug = true;
+        } else if (argValue === '-ih' || argValue === '--ignorehooks') {
```

Comment:
We follow POSIX conventions, which should be spelled `--ignore-hooks` instead of `--ignorehooks`

Reply:
Having a multiple character short form is a bit odd. In most POSIX applications, short flags are single characters and can be combined into a single expression, like `-dh` being equivalent to `-d -h`.